### DISCUSSION TITLE
Store and restore scroll position?

### DISCUSSION THREAD

**Discussion by rconstantine** [quasarframework/quasar#10245] [CLI - PWA mode] [answered]
I’m serving content from a website to a PWA. I save the pages visited to the user’s device and when the user returns to the app I reload the last page they viewed. Is there a way to store how far down they were and reload the position and auto scroll to it?

**Comment by pdanpdan** [comment, marked as answer]
https://router.vuejs.org/guide/advanced/scroll-behavior.html
https://github.com/vuejs/vue-router/blob/dev/examples/scroll-behavior/app.js
